src/widgets/ot_pulltorefresh.eliom

```diff
@@ -0,0 +1,261 @@
+[%%shared.start]
+
+[%%client 
+  open Eliom_content.Html
+]
+
+open Eliom_content.Html.D
+
+[%%client
+
+  module type PULLTOREFRESH = sig
+    val dragThreshold : float 
+    val moveCount : int
+    val headContainerHeight : int
+    val pullDownIcon : Html_types.div  Eliom_content.Html.D.elt
+    val loadingIcon : Html_types.div  Eliom_content.Html.D.elt
+    val successIcon : Html_types.div  Eliom_content.Html.D.elt
+    val failureIcon : Html_types.div  Eliom_content.Html.D.elt
+    val pullText :  Html_types.span  Eliom_content.Html.D.elt 
+    val container :  Html_types.div  Eliom_content.Html.D.elt
+    val pullDownText : string
+    val releaseText : string
+    val loadingText : string
+    val successText : string
+    val failureText : string
+    val rotateGradually : bool
+    val blockPullIcon : bool
+    val afterPull : unit -> bool Lwt.t
+  end
+
+  module Make(Elt:PULLTOREFRESH) = struct
```

Comment:
I'd prefer the name `Conf` to `Elt`. It is a configuration that is supplied as an argument not a DOM element.